lib/graphql/account_settings/cloud_manager.dart

```diff
@@ -0,0 +1,146 @@
+import 'dart:async';
+
+import 'package:autonomy_flutter/common/injector.dart';
+import 'package:autonomy_flutter/database/cloud_database.dart';
+import 'package:autonomy_flutter/gateway/iap_api.dart';
+import 'package:autonomy_flutter/graphql/account_settings/account_settings_client.dart';
+import 'package:autonomy_flutter/graphql/account_settings/account_settings_db.dart';
+import 'package:autonomy_flutter/graphql/account_settings/cloud_object/address_cloud_object.dart';
+import 'package:autonomy_flutter/graphql/account_settings/cloud_object/connection_cloud_object.dart';
+import 'package:autonomy_flutter/service/account_service.dart';
+import 'package:autonomy_flutter/service/configuration_service.dart';
+import 'package:autonomy_flutter/service/settings_data_service.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+class CloudManager {
+  late final String _requester;
+  late final String _deviceId;
+  late final String _flavor;
+
+  late final WalletAddressCloudObject _walletAddressObject;
+  late final AccountSettingsDB _addressAccountSettingsDB;
+
+  late final ConnectionCloudObject _connectionObject;
+  late final AccountSettingsDB _connectionAccountSettingsDB;
```

Comment:
why we still need this `_connectionAccountSettingsDB ` and `_addressAccountSettingsDB `? It should be accessed from `_connectionObject ` and `_walletAddressObject `

Reply:
fixed